parser/main.py

```diff
@@ -101,6 +104,9 @@
     SECRET_KEY: "admin"
 }
 
+# <----- Format Specificer List for all Data Types ----->
+format_specifier_list = [CAR_DBC]
```

Comment:
Not super clear why the variable is named this way.

Reply:
Additionally, the idea of a "DBC" should be abstracted away from the main.py of the parser, and should only live in the CAN message datatype.

Reply:
Ah that makes much more sense. I have now removed the format specifier from main.py, link_telemetry, and other places and kept it so that it is abstracted to the data class implementation.

Thanks!

As an aside, the idea of the format_specifer_list was to put any files or objects that can parse your message in a list that is passed to the create_message factory method that will essentially figure out what message type to return back.

Reply:
I made another change here to replace the format specifier list in every single class. Instead, there is just a file called `parameters.py` that holds the DBC_FILE, CAR_DBC, and other constants like the min and max lengths of the messages. I then just import the constants I need into the files that need them.